Demo/KommunicateChatUI-iOS-SDK-Demo.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We can remove this file. I don't think it's required.

Reply:
@mukeshthawani  done